doc/getting-started/connections.rst

```diff
@@ -0,0 +1,76 @@
+Connections
+===========
+
+PyVista is a powerful tool that researchers can harness to create compelling,
+integrated visualizations of large datasets in an intuitive, Pythonic manner.
+
+PyVista is used across science and engineering disciplines by a diverse
+community to visualize data/models in 3D, generate publication-quality
+illustrations, automate analysis workflows, and build custom applications
+leveraging PyVista's 3D capabilities.
+
+Here are a few open-source projects that leverage PyVista:
+
+
+Engineering
+-----------
+
+* pyleecan_: Electrical engineering open-source software providing a user-friendly, unified, flexible simulation framework for the multiphysic design and optimization of electrical machines and drives.
+* pymapdl_: Pythonic interface to Ansys MAPDL.
+* pyprocar_: A Python library for electronic structure pre/post-processing.
+* AeroSandbox_: Plug-and-play analysis tools for aerodynamics, propulsion, structures, trajectory design, and much more.
+* Ptera_: Software for nalyzing flapping-wing flight.
+
+
+Geosciences
+-----------
+
+* GemPy_: 3D structural geological modeling software using PyVista for 3D visualization and interaction.
+* gemgis_: Spatial data processing for geomodeling.
+* gimli_: Geophysical inversion and modeling library.
+* discretize_: Discretization tools for finite volume and inverse problems.
+* SEGY-SAK_: SEGY Swiss Army Knife for Seismic Data.
+* subsurface_: Core data exchange library for subsurface science and engineering
+* Flownet_: Data-Driven Reservoir Predictions.
+* PVGeo_: Python package of VTK-based algorithms to analyze geoscientific data and models. PyVista is used to make the inputs and outputs of PVGeo's algorithms more accessible.
+* omfvista_: 3D visualization for the Open Mining Format (omf). PyVista provides the foundation for this library's visualization.
```

Comment:
I'm thinking of trimming these out... 😢

Reply:
These other packages, GemPy, gimli, SimPEG's discretize, etc. are in a whole other league

Reply:
> I'm thinking of trimming PVGeo and omfvista out

I'm happy with keeping them in, and having a long long list of connected software is great. By categorizing, we can have several projects per category still and it's nonetheless great.